### PR TITLE
implement remaining commands

### PR DESCRIPTION
**Pull request by felixn**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

**Comment by felixn** [comment]
We now have all commands from the XCP 1.5 base standard implemented!

**Comment by christoph2** [comment]
Excellent Felix! 
So it's up to me to implement the missing features (mainly error-handlig).

File: pyxcp/master/base.py [hunk 1]
```diff
@@ -170,6 +173,20 @@ def connect(self):
         self.DWORD_pack = makeDWordPacker(byteOrderPrefix)
         self.WORD_unpack = makeWordUnpacker(byteOrderPrefix)
         self.DWORD_unpack = makeDWordUnpacker(byteOrderPrefix)
+
+        if self.slaveProperties.addressGranularity == \
+                types.AddressGranularity.BYTE:
+            self.AG_pack = struct.Struct("<B").pack
+            self.AG_unpack = struct.Struct("<B").pack
```
Comment (waszil) [inline pyxcp/master/base.py]: shouldn't this be struct.Struct("<B").**un**pack?

Comment (felixn) [reply pyxcp/master/base.py]: Yup, good catch!
I'll fix it with a future pull request.

**Pull request merged**